Jenkinsfile

```diff
@@ -55,7 +55,7 @@ pipeline {
       steps {
         echo 'Deploying to Staging'
         sh "curl --upload-file staging/target/metadata-ws-*.war \
-        'http://'${tomcatCredentials}'@'${stagingHost}':8080/manager/text/deploy?path=/metadata&update=true' | \
+        'http://'${tomcatCredentials}'@'${stagingHost}':8080/manager/text/deploy?path=/dev/metadata&update=true' | \
```

Comment:
Shouldn't we in proxy settings map `/dev/metadata` to `http://staging:8080/metadata`?

Reply:
the Swagger base path takes host + prefix + servletContext only. So the if we deploy using /metadata - the staging app base url forms to ebi.ac.uk+ ega/ampt2d + metadata - which is again prod app base url.

Reply:
Ok! Then I am fine with this change.